MCprep_addon/mcprep_ui.py

```diff
@@ -327,12 +358,31 @@ def draw(self, context):
 			col.label(text="Meshwap assets")
 			col = split.column()
 			col.prop(self, "meshswap_path", text="")
+			
 			if not os.path.isfile(bpy.path.abspath(self.meshswap_path)):
 				row = box.row()
 				row.label(text="MeshSwap file not found", icon="ERROR")
 
+			split = util.layout_split(box, factor=factor_width)
+			col = split.column()
+			col.label(text="Model assets")
+			
+			col = split.column()
+			col.prop(self, "model_path", text="")
+			if not os.path.isfile(bpy.path.abspath(self.model_path)):
+				row = box.row()
+				row.label(text="Model file not found", icon="ERROR")
+	
+
+			
+
+
+
+
 			row = layout.row()
 			row.scale_y=0.7
+
```

Comment:
nit pick: unnecessary white spaces/newlines. Optional to update, but generally I keep 1 newline between functions within a class (e.g. within a single operator definition), and 2 lines between the top-level modules (e.g. 2 lines between the end of one class to the next, or between functions defined on their own at the module level). Only exception is sometimes I use three between high level sections (e.g. where I separate operator definitions from registration; and technically isn't totally pythonic but I find improves readability)